rscripts/YAL5-single-codons.R

```diff
@@ -0,0 +1,1108 @@
+# This script uses a h5, GFF and .tsv file to create a metafeature plot
```

Comment:
Could also mention here that the script can also generate tsv table with feature_of_interest and RelCount when there are multiple features